packages/components/src/Enumeration/Items/Item/Item.component.js

```diff
@@ -47,7 +47,9 @@ function Item({ id, item, searchCriteria, showCheckboxes, style, t }) {
 				});
 			}
 		}
-
+		if (action.disabled) {
+			return null;
+		}
```

Comment:
Same remark : You should filter the actions in the main component return, before the map()